README.md

```diff
@@ -31,3 +31,31 @@
     - You can edit the system message the bot uses, or disable it entirely
 8. Start the bot with `npm start`
 9. You can interact with the bot by @mentioning it with your message
+
+### Set-up instructions Docker
+1. Install [docker](https://docs.docker.com/get-docker/)
+   - should be atleast compatible with version 3 of compose (docker engine 1.13.0+)
+2. Install [Ollama](https://github.com/jmorganca/ollama) (ditto)
```

Comment:
ditto means same thing as before, doesn't make sense here, put "if you have a package manager, use that instead to install this"

Reply:
Sounds good and good catch :+1:

Reply:
This should be resolved in https://github.com/mekb-turtle/discord-ai-bot/pull/13/commits/e7e087d87819f01ebda8d8b1a098bc4d4ad19451